model/wire.cc

```diff
@@ -363,6 +363,19 @@ namespace
     }
     cairo_path_destroy (path);              
   }
+  
+  bool Wire::attachedToDefiningVar() const
+  {
+    auto f=from(), t=to();
+    assert(f && t);
+    bool attachedToFrom=false, attachedToTo=false;                
+    if (auto vf=f->item().variableCast()) attachedToFrom=vf->varTabDisplay;
+    //else return f->item().attachedToDefiningVar();                                // attempt at recursion...
```

Comment:
See my note above. If Variable::attachedToDefiningVar() is an overridden implementation, this implementation can just be the result of to()->attachedToDefiningVar(). I don't think the status of the from() item is relevant to determine visibility on the canvas, only the to() item..

Reply:
Thank you. Yes, it doesn't make sense to include from() here :)